pkg/kritis/metadata/grafeas/config.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package grafeas
+
+import (
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+// file is the Grafeas configuration file.
+type file struct {
+	GrafeasCerts *CertConfig `yaml:"grafeascerts"`
+}
+
+// CertConfig is the Grafeas certificates configuration.
+type CertConfig struct {
+	CertFile string `yaml:"certfile"` // A PEM encoded certificate file
+	KeyFile  string `yaml:"keyfile"`  // A PEM encoded private key file
+	CAFile   string `yaml:"cafile"`   // A PEM encoded CA's certificate file
+}
+
+// defaultConfig is a configuration that can be used as a fallback value.
+func defaultConfig() *CertConfig {
+	return &CertConfig{
+		CertFile: "",
+		KeyFile:  "",
+		CAFile:   "",
+	}
```

Comment:
FYI - this should be equivalent to return &CertConfig{} (and in fact you may not need the default function at all).

Reply:
Good point, done.